frontend/src/models/artist.ts

```diff
@@ -0,0 +1,19 @@
+export type Artist = {
+  followers: number;
+  genres: string[];
+  mainArtist: string;
+  featuredArtists: string[];
+  popularity: number;
+  images: string; // TODO why did this used to be Array<Map<string, any>>
```

Comment:
ya this is a bug on my end that i thought i fixed, it should be an image[] where image is an object like:

`image = {
url: string;
height: number;
width: number;
}`